src/Core/Services/Implementations/GroupService.cs

```diff
@@ -61,7 +61,16 @@ public async Task SaveAsync(Group group, IEnumerable<SelectionReadOnly> collecti
         else
         {
             group.RevisionDate = DateTime.UtcNow;
-            await _groupRepository.ReplaceAsync(group, collections ?? new List<SelectionReadOnly>());
+
+            if (collections == null)
```

Comment:
I worry that this may have an impact on group saving somewhere else when collection associations are not passed in. Maybe not.

Reply:
I thought about that but if they're not passed then they would be defined as a new list.